readthedocs/oauth/management/commands/sync_vcs_data.py

```diff
@@ -0,0 +1,74 @@
+from django.contrib.auth.models import User
+from django.core.management.base import BaseCommand
+
+from readthedocs.oauth.tasks import sync_remote_repositories
+
+
+class Command(BaseCommand):
+    help = "Sync OAuth RemoteRepository and RemoteOrganization"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--queue',
+            type=str,
+            default='resync-oauth',
+            help='Celery queue name.',
+        )
+        parser.add_argument(
+            '--users',
+            nargs='*',
+            type=str,
+            default=[],
+            help='Re-sync VCS provider data for specific users only.',
+        )
+        parser.add_argument(
+            '--skip-users',
+            nargs='*',
+            type=str,
+            default=[],
+            help='Skip re-sync VCS provider data for specific users.',
+        )
+        parser.add_argument(
+            '--max-users',
+            type=int,
+            default=100,
+            help='Maximum number of users that should be synced.',
+        )
+
+    def handle(self, *args, **options):
+        queue = options.get('queue')
+        sync_users = options.get('users')
+        skip_users = options.get('skip_users')
+        max_users = options.get('max_users')
+
+        # Filter users who have social accounts connected
+        # and has no remote repository relations
+        users = User.objects.filter(
+            socialaccount__isnull=False,
+            remote_repository_relations__isnull=True
+        ).distinct()
```

Comment:
I think we shouldn't apply these filters if we pass `sync_users` so we can force a re-sync for a user that was already synced some time ago.

Or... maybe another parameter `--force` that avoid these two filters is more explicit here.

Reply:
@humitos Shouldn't we keep `socialaccount__isnull=False` for all the filters? As there is no point trying to sync and triggering a task for a user who doesn't have any social account connected.

Reply:
Yeah, I didn't express myself correctly. Only the `remote_repository_relations__isnull=True` is the one that I wanted to remove.